crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -538,7 +538,7 @@ pub enum Operation {
     Jeq(NodeId, BlockId), //jump on equal
     Jmp(BlockId),         //unconditional jump
     Phi { root: NodeId, block_args: Vec<(NodeId, BlockId)> },
-    Call(noirc_frontend::node_interner::FuncId, Vec<NodeId>, Vec<ArrayId>), //Call a function
+    Call(noirc_frontend::node_interner::FuncId, Vec<NodeId>, Vec<(ArrayId, u32)>), //Call a function
```

Comment:
This vector contains the list of arrays returned by the function call. But because the function can return also other types (not only arrays), the u32 indicates which return value it is (the first one, or the second one..)